yalla-coop/website-redesign
# Work Card

**Issue**: #22 

## What did I do?
Created the work card and another component to handle images :

- [x] Built the work card
- [x] Styled the work card
- [x] Build a component to handle images

I've created this image component so we don't have to write a graphql query to get the transformed image that we give to the gatsby image. The component takes two props, **fileName** (the name of the image in the images folder) and **altText**

## How to test?
I couldn't do a storybook representation for the component since storybook can't see graphql local API but I can give you a code to put in any page so you can try out the work card component

```jsx
   <div
      css={css`
        width: 40%;
        margin: 2rem;
      `}
    >
      <WorkCard
        smallHeading="this is a small heading"
        bigHeading="this is a big kind of heading for bigger text"
        fileName="mask-group"
        altText="mask group"
      />
    </div>
```
Closes #22
